.coderabbit.yml

```
@@ -0,0 +1,252 @@
+# Code Rabbit Configuration for Starbunk Discord Bot Project
+# Optimized for TypeScript/Node.js containerized architecture
+
+# Review settings
+reviews:
+  # Enable high-quality reviews with focus on security and performance
+  high_level_summary: true
+  poem: false
+  review_status: true
+  collapse_empty_files: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)